### ISSUE TITLE
Carbon v9 TableToolbarSearch shows accessibility issues while running with DAP tool

### ISSUE THREAD

**Issue by SowmyaRaviKrishnan** [carbon-design-system/carbon#4642]
Carbon v9 TableToolbarSearch shows below accessibility issues while running with DAP tool
1. INPUT LABEL references must be valid
2.The WAI-ARIA widget with IMG role must have a nonempty alternate text or a label.
3.An interactive element/widget must have an accessible name.

## Environment

> Operating system
Linux - RHEL

> Browser
Chrome

> Assistive technology used to verify
IBM DAP

## Detailed description

> What version of the Carbon Design System are you using?
V9

> What WCAG 2.1 checkpoint does the issue violate?
Checkpoint 4.1.2 Name, Role, Value

**Comment by SowmyaRaviKrishnan** [comment]
@joshblack & @amuqeetkhan
Issue raised from IBM Secure Service Container Development team India

**Comment by abbeyhrt** [comment]
Hi @SowmyaRaviKrishnan, thanks for bringing this to our attention! As I've mentioned in the other issues for v9, we will not be backporting WCAG 2.1 fixes to v9 but we always accept community contributions for any v9 issues! If you would like this to be addressed, feel free to open up a PR and we’ll review it! 👍
